bot/seasons/evergreen/bookmark.py

```diff
@@ -46,6 +63,25 @@ def __init__(self, bot: commands.Bot):
         else:
             await ctx.message.add_reaction(Emojis.envelope)
 
+        embed.add_field(
+            name=f'Bookmarked from {ctx.author.name}.',
+            value=f'[Visit original message]({ctx.message.jump_url})',
+            inline=False
+        )
+
+        def check(reaction: str, user: str) -> None:
+            return str(reaction.emoji) == Emojis.envelope
+
+        sent_person = [ctx.author.id]  # list of id who got the message

```

Comment:
`Member` instances already support direct comparison, there's no need to use IDs.